tests/unit/plugins/inventory/test_kubevirt.py

```diff
@@ -786,12 +775,12 @@ def inventory_groups_create_groups_option(inventory_groups):
 
 
 @pytest.fixture(scope="module")
-def base_vmi_host_vars():
+def empty_host_vars():
```

Comment:
Pass something like `None` instead and check in the test that when `None` then it should be empty? Saves adding this fixture.

Reply:
I would like to keep it as it is. While it is true that we are adding a "silly" fixture, IMHO the benefit is that it is really clear from the parametrize mark that the inventory should be empty without making the code harder to read (no if else statements required) the test code more.

Reply:
It's okay in this case, but we really need to keep `test_kubevirt.py` maintainable.